v19.2/use-the-built-in-sql-client.md

```diff
@@ -74,6 +74,7 @@ Flag | Description
 <a name="sql-flag-format"></a> `--format` | How to display table rows printed to the standard output. Possible values: `tsv`, `csv`, `table`, `raw`, `records`, `sql`, `html`.<br><br>**Default:** `table` for sessions that [output on a terminal](#session-and-output-types); `tsv` otherwise<br /><br />This flag corresponds to the `display_format` [client-side option](#client-side-options).
 `--safe-updates` | Disallow potentially unsafe SQL statements, including `DELETE` without a `WHERE` clause, `UPDATE` without a `WHERE` clause, and `ALTER TABLE ... DROP COLUMN`.<br><br>**Default:** `true` for [interactive sessions](#session-and-output-types); `false` otherwise<br /><br />Potentially unsafe SQL statements can also be allowed/disallowed for an entire session via the `sql_safe_updates` [session variable](set-vars.html).
 `--set` | Set a [client-side option](#client-side-options) before starting the SQL shell or executing SQL statements from the command line via `--execute`. This flag may be specified multiple times, once per option.<br><br>After starting the SQL shell, the `\set` and `unset` commands can be use to enable and disable client-side options as well.  
+`--watch` | Repeat the SQL commands specified with `--execute` or `-e` until a SQL error occurs or the process is terminated. You must also specify an interval at which to repeat the statement. Valid [time units](https://en.wikipedia.org/wiki/Orders_of_magnitude_(time)) are `ns`, `us`, `ms`, `s`, `m`, and `h`. See the [example](#repeat-a-sql-statement) below.<br /><br /> Note that this flag is intended for simple monitoring scenarios during development and testing.
```

Comment:
1) it's possible to have multiple uses of `-e`; the `--watch` applies to all of them not just one

2) I would not put a hyperlink for the units and instead phrase it as: "... specify a time interval between executions of the statements; a time interval is a number followed by a time unit, for example 's' for seconds". Let the user figure out the other units on their own.

(We do not have a fixed set of units today, we're dependent on another package for this, and we probably don't want to commit on a particular set of units. I'd say just `s` is guaranteed.)